visualization/app/codeCharta/util/reduxHelper.ts

```diff
@@ -1,9 +1,9 @@
 import { clone } from "./clone"
-import isEqual from "lodash.isequal"
+import { dequal } from "dequal/lite"
```

Comment:
I understand that we currently only use very this function in a very limited way and lite would here be sufficient. This could in theory change over time though and e.g., `removeItemFromArray()` could handle data that does contain more complex data. As such, we should always use the strictest possible comparison algorithm possible as it is the only way to guarantee that things properly work.
The only case where the lite mode would be safe to use is in case only JSON data is used and that's not the case.

Reply:
currently `removeItemFromArray` is in `reduxHelper` and all redux data should be plain JSON data by idea of redux design. But I guess these comparison are currently no bottleneck, so lets be better safe than sorry - adjusted.